src/content/reference/react/useTransition.md

```diff
@@ -1501,6 +1502,92 @@ main {
 
 ---
 
+### Displaying an error to users with a error boundary {/*displaying-an-error-to-users-with-error-boundary*/}
+
+<Canary>
+
+Error Boundary for useTransition is currently only available in React's canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+If you'd like to display an error to your users when using `useTransition`, you can use an [error boundary](/reference/react/Component#catching-rendering-errors-with-an-error-boundary). To use an error boundary, wrap the component where you are calling the `useTransition` in an error boundary. If the function in `startTransition` is rejected the fallback for the error boundary will be displayed.
```

Comment:
I think "rejected" is a term typically just used with Promises. "throws" would be more general.

Reply:
Technically since it's an async function it is a promise that rejects after the first `await`, right? Not sure the clearest way to say this.

Reply:
"throws" feels right to me for async functions as you can `catch` the error around the `await`. Just a feeling though!